hip-0000-zero-token-operations.md

```diff
@@ -0,0 +1,70 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Zero unit token operations in smart contracts
+author: littletarzan, vae.vecturne
+working-group:
+type: Standards track
+category: Service
+needs-council-approval: No
+status: Draft
+created: Sept 1, 2022
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/563
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+Integer rounding will naturally lead to DeFi contracts sometimes minting or transferring 0 units of a token. But the network rejects 0 units in these operations, forcing contract developers to include special cases that would not be necessary on most networks.
+
+## Motivation
+
+To reduce code, if statement checks, etc inside smart contracts and make DeFi contracts easier to use. To reduce bugs in DeFi applications involving math and rounding.
```

Comment:
Perhaps we can flesh this out a bit?
From [hip-1](https://hips.hedera.com/hip/hip-1):
The motivation should clearly explain why the existing specification is inadequate to address the problem that the HIP solves.